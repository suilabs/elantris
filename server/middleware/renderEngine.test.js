import fs from 'fs';

import render from './renderEngine';

describe('RenderEngine', () => {
  it('returns string with substitued vars', async () => {
    const str = '<h1>{{{title}}</h1>';
    const params = {
      title: 'this title',
    };
    const filePath = '/tmp/test.html';
    fs.writeFileSync(filePath, str);
    const resp = await render(filePath, params);
    expect(resp).toMatchSnapshot();
  });
});
