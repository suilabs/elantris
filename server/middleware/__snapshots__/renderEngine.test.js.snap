// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['RenderEngine returns string with substitued vars 1'] = '"<h1>{{{title}}</h1>"';
